conf/st

```diff
@@ -53,6 +53,8 @@ M0_SRC_DIR=${M0_SRC_DIR%/*/*}
 
 M0CONFGEN=$M0_SRC_DIR/utils/m0confgen
 
+XPRT=$(m0_default_xpt)
```

Comment:
Sorry, I didn't see where the variable or function "m0_default_xpt" is defined.

Is it defined by command line argument?
or environment variable?

Please add comments to say how it is defined, and how people can change it.